etc/taskcluster/decision_task.py

```diff
@@ -393,8 +393,8 @@ def windows_arm64():
         windows_build_task("UWP dev build", arch="arm64", package=False)
         .with_treeherder("Windows arm64")
         .with_script(
-            "python mach build --dev --uwp --win-arm64",
```

Comment:
Are the `--uwp` and `--win-arm64` flags still useful, or should they be removed from mach?

Reply:
I think it's time to remove them.